IdokladSdk.IntegrationTests/Tests/Clients/ProformaInvoice/ProformaInvoiceTests.Async.cs

```diff
@@ -16,34 +16,33 @@ namespace IdokladSdk.IntegrationTests.Tests.Clients.ProformaInvoice
     /// </summary>
     public partial class ProformaInvoiceTests
     {
-        private ProformaInvoicePostModel _proformaInvoicePostModelAsync;
```

Comment:
Proč to bylo před tím uložené ve fieldu? Používalo se to na více místech?

Reply:
nie som si úplne istý prečo to tak bolo, ale podľa toho ako sa to teraz používa, to bolo zbytočné, tak som to prerobil